packages/web-runtime/src/services/uppyService.ts

```diff
@@ -0,0 +1,223 @@
+import Uppy from '@uppy/core'
+import { CustomTus } from '../composables/upload/uppyPlugins/customTus'
+import XHRUpload, { XHRUploadOptions } from '@uppy/xhr-upload'
+import { CustomDropTarget } from '../composables/upload/uppyPlugins/customDropTarget'
+import StatusBar from '@uppy/status-bar'
+import { UppyResource } from '../composables/upload'
+import Vue from 'vue'
+
+export class UppyService extends Vue {
```

Comment:
We used this as a quick hack, to port away from events on the `$root` instance. Do we have a better provider for the `$on`, `$off` and `$emit` functionality. EventBus maybe?